### PR TITLE
Feature/homing for new fdm

### PR DESCRIPTION
**Pull request by ipa-fmw**

### CODE REVIEW

**Comment by ipa-mdl** [comment]
do you have a scetch?

**Comment by ipa-mdl** [comment]
What is the offset of the homing switch to the axis?

File: cob4_homing/scripts/calib.py [hunk 1]
```diff
@@ -16,10 +16,18 @@
...
+#    ('please align wheels 3 and 1', -90000, (WHEEL3, -30000), (WHEEL1, -150000)),
+#}
+
+# new FDMs
 tests = {
-    ('please align wheels 1 and 2',  30000, (WHEEL1, -30000), (WHEEL2, -150000)),
-    ('please align wheels 2 and 3', 150000, (WHEEL2, -30000), (WHEEL3, -150000)),
-    ('please align wheels 3 and 1', -90000, (WHEEL3, -30000), (WHEEL1, -150000)),
+    ('please align wheels 1 and 2', -150000, (WHEEL1, -120000), (WHEEL2, 120000)),
+    ('please align wheels 2 and 3',  -30000, (WHEEL2, -120000), (WHEEL3, 135000)),
```
Comment (ipa-mdl) [inline cob4_homing/scripts/calib.py]: Your global angles are fine, they actually make more sense as they correspond to the vectors.
However I am not sure about the `135000`.

**Comment by ipa-fxm** [comment]
how is this related to https://github.com/ipa320/cob_robots/pull/677?

**Comment by ipa-mdl** [comment]
@ipa-fxm: not related at all (?)

**Comment by ipa-fmw** [comment]
not related at all.

in https://github.com/ipa320/cob_robots/pull/677 we have a test for a single FDM. This PR handles the calibration of 3 FDMs on an assembled base.

**Pull request merged**